app/models/bookable_slot.rb

```diff
@@ -31,23 +31,27 @@ def self.bookable
       .without_holidays
   end
 
-  def self.without_appointments
+  def self.without_appointments # rubocop:disable Metrics/MethodLength
     joins(<<-SQL
             LEFT JOIN appointments ON
-              appointments.guider_id = #{quoted_table_name}.guider_id AND
-              appointments.start_at = #{quoted_table_name}.start_at AND
-              appointments.end_at = #{quoted_table_name}.end_at AND NOT
-              appointments.status IN (
+              appointments.guider_id = #{quoted_table_name}.guider_id
+              AND NOT appointments.status IN (
                 #{Appointment.statuses['cancelled_by_customer']},
                 #{Appointment.statuses['cancelled_by_pension_wise']}
               )
+              AND (
+                (appointments.start_at = #{quoted_table_name}.start_at AND appointments.end_at = #{quoted_table_name}.end_at)
+                OR
+                (appointments.start_at BETWEEN #{quoted_table_name}.start_at AND #{quoted_table_name}.end_at)
+                OR
+                (appointments.end_at BETWEEN #{quoted_table_name}.start_at AND #{quoted_table_name}.end_at)
+              )
             SQL
```

Comment:
Believe this would achieve the same thing:
```
AND (
(appointments.start_at < #{quoted_table_name}.end_at 
AND appointments.end_at > #{quoted_table_name}.start_at)
)
```